olm-catalog/serverless-operator/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM scratch
+
+COPY manifests /manifests
+COPY metadata/annotations.yaml /metadata/annotations.yaml
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=serverless-operator
+LABEL operators.operatorframework.io.bundle.channel.default.v1="4.3"
+LABEL operators.operatorframework.io.bundle.channels.v1="4.3,4.4,4.5"
+
+LABEL \
+      com.redhat.component="openshift-serverless-1-serverless-rhel8-operator-container" \
+      name="openshift-serverless-1/serverless-rhel8-operator" \
+      version="1.7.3" \
+      summary="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      maintainer="serverless-support@redhat.com" \
+      description="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      io.k8s.display-name="Red Hat OpenShift Serverless 1 Serverless Operator" \
+      com.redhat.openshift.versions="v4.5" \
```

Comment:
This should probably not be 4.5 (see my previous comment on versions)

Reply:
As I understand it, this refers to the OCP version.

Reply:
I'm still wondering why this line has "v4.5". IMO, this line should list the same versions as `operators.operatorframework.io.bundle.channels.v1` above. For example, Serverless 1.10 had this: https://github.com/openshift-knative/serverless-operator/blob/release-1.10/olm-catalog/serverless-operator/Dockerfile
According to https://docs.google.com/spreadsheets/d/1HTxR37_MM03_JQImW-KmAFk1oEQ83fXzfGxdfWQSX6o/edit#gid=0 , Serverless 1.7.3 will be supported on 4.3, 4.4 ( but it must also work on 4.5 because that's the upgrade path to 1.8).

Reply:
Alright, https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Serverless mentions only 4.4 as the supported version. So I think this line should have "v4.4" and the channels above should list only "4.4"

Reply:
@warrenvw @mgencur 

This is the OCP version. Here is the list of supported versions: https://docs.engineering.redhat.com/display/CFC/Delivery

"v4.5" includes <= 4.4 because everything pushed to the 4.5 index gets backported to app-registry and served to all clusters <= 4.5. In a couple weeks you may be able to use the new version scheme in the labels and have the label be set to "=v4.5" which will prevent these versions from being pulled forward to the 4.6 and 4.7 index images. Today, this new versioning scheme is not available and you can instead mark operator versions as deprecated using the flow described in the above doc.